server.go

```diff
@@ -21,6 +21,10 @@ const (
 type Server struct {
 	Config        ServiceProviderConfig
 	ResourceTypes []ResourceType
+
+	// GetResourcePath optional function that strips the path prefix. If not provided we will assume the path
+	// prefix is /v2. This is generally used to support tenancy. Example: "/scim/:tenantId/v2/Users" => "/Users"
+	GetResourcePath func(path string) string
```

Comment:
I don't see why this is necessary. The `/v2` part is the only required prefix according to the RFC. If users need an additional stripping step, they should implement it themselves IMHO (for example by placing a `http.StripPrefix` in front of the server). The complete URI is available in the request parameter anyway.

Reply:
@jdeflander the handlers need to know about the tenant id that is part of the url structure, I am not sure if there is a way to strip the prefix after adding the tenantId to request context, I would definitely prefer that if so.

Reply:
Wouldn't it be possible to inspect the `URL` field of the request that is passed to your handlers?

Reply:
Hmmm maybe I don’t know how strip prefix works. Let me dig into that when I get to my computer and report back.